src/components/editor/EditorAppController.jsx

```diff
@@ -52,6 +81,7 @@ export default class EditorAppController extends BaseComponent {
         }
       });
     });
+    this.isrestarted();
```

Comment:
This should be inside the `on 'end'` handler, you don't want to start pinging before you received the start message. Put this in the `reply === 'start'` conditional.